spinalcordtoolbox/vertebrae/core.py

```diff
@@ -142,11 +153,30 @@ def vertebral_detection(fname, fname_seg, contrast, param, init_disc, verbose=1,
     # create list for z and disc
     list_disc_z = []
     list_disc_value = []
-    zrange = list(range(-10, 10))
+    zrange = list(range(-9, 10))
```

Comment:
it does. @lrouhier any reason for changing this?

Reply:
I will check that it is definitely better like this (as opposed to (-10,10) ). But I think I changed it because a bug I had that is now fixed. I will get back to you on this.